src/lib/snack-bar/snack-bar.ts

```diff
@@ -144,7 +144,33 @@ export class MdSnackBar {
   private _createOverlay(config: MdSnackBarConfig): OverlayRef {
     let state = new OverlayState();
     state.direction = config.direction;
-    state.positionStrategy = this._overlay.position().global().centerHorizontally().bottom('0');
+    
+    let positionStrategy = this._overlay.position().global();
+    // TODO(josephperrott): Add 'start' and 'end' to handle directionality after #4044 merges.
```

Comment:
Can still have the code for ltr/rtl now; you can just use the direction from the config for now